README.md

```diff
@@ -159,6 +162,19 @@ Of type `u64`, initialized to `0`.
 
 Expresses the portion of Surplus Weight which has been refunded. Not used on XCM platforms which do not require payment for execution.
 
+### **3.10** Transact Status
+
+Of type `MaybeErrorCode`, initialized to `MaybeErrorCode::Success`.
+
+Expresses the status of a `Transact` instruction after the encoded call within has been dispatched.
```

Comment:
It doesn't express the status, but the result of it.

Reply:
Changed.